### PR TITLE
I aM Utterly confused

### PR DESCRIPTION
**Pull request by ActuallyTaylor**

### CODE REVIEW

**Comment by mjmagee991** [comment]
[IMU CANopen Setup](https://trello.com/c/YtiXs36X/128-imu-canopen-setup)

File: include/IMU.hpp [hunk 1]
```diff
@@ -1,13 +1,340 @@
...
+         .Type = nullptr,
+         .Data = (uintptr_t) 0xFE},
+        {//no inhibit time
+         .Key = CO_KEY(0x1800, 3, CO_UNSIGNED16 | CO_OBJ_D__R_),
+         .Type = nullptr,
+         .Data = (uintptr_t) 0},
+        {//send every 2 seconds
+         .Key = CO_KEY(0x1800, 5, CO_UNSIGNED16 | CO_OBJ_D__R_),
+         .Type = CO_TEVENT,
+         .Data = (uintptr_t) 2000},
```
Comment (mjmagee991) [inline include/IMU.hpp]: Add commas to fix the formatting

File: include/IMU.hpp [hunk 2]
```diff
@@ -1,13 +1,340 @@
...
+            .Data = (uintptr_t) 0x600 + NODE_ID,
+        },
+        {
+            .Key = CO_KEY(0x1200, 2, CO_UNSIGNED32 | CO_OBJ_D__R_),
+            .Type = nullptr,
+            .Data = (uintptr_t) 0x580 + NODE_ID,
+        },
+
+        /**
+         * TPDO 1 Settings
```
Comment (mjmagee991) [inline include/IMU.hpp]:
```suggestion
         * TPDO 0 Settings
```

File: src/IMU.cpp [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
-namespace IMU {}
+namespace IMU {
+CO_OBJ_T* IMU::getObjectDictionary() {
+    return objectDictionary;
+}
+
+uint16_t IMU::getObjectDictionarySize() const {
+    return OBJECT_DICTIONARY_SIZE;
+}
+}// namespace IMU
```
Comment (mjmagee991) [inline src/IMU.cpp]: Add a newline

File: targets/DEV1-IMU/main.cpp [hunk 4]
```diff
@@ -1,29 +1,179 @@
 /**
- * This is a basic sample of using the UART module. The program provides a
- * basic echo functionality where the uart will write back whatever the user
- * enters.
- */
+* This is a basic sample of using the UART module. The program provides a
```
Comment (mjmagee991) [inline targets/DEV1-IMU/main.cpp]: Update this comment

File: targets/DEV1-IMU/main.cpp [hunk 5]
```diff
@@ -1,29 +1,179 @@
...
+        CONodeProcess(&canNode);
+        // Update the state of timer based events
+        COTmrService(&canNode.Tmr);
+        // Handle executing timer events that have elapsed
+        COTmrProcess(&canNode.Tmr);
+        // Wait for new data to come in
+        time::wait(100);
     }
-}
+}
```
Comment (mjmagee991) [inline targets/DEV1-IMU/main.cpp]: Add a newline

**Review by mjmagee991 — CHANGES_REQUESTED** [review]
Just a couple formatting issues

**Review by mjmagee991 — APPROVED** [review]
Nice work

**Review by Ethangirard62 — APPROVED** [review]

**Pull request merged**